packages/vitest/src/plugins/mock.ts

```diff
@@ -87,7 +88,7 @@ const getMethodCall = (method: string, actualPath: string, importPath: string) =
   if (actualPath.includes('/node_modules/'))
     nodeModule = `"${importPath}"`
 
-  return `__vitest__${method}__("${actualPath}", ${nodeModule}`
+  return `await __vitest__${method}__("${actualPath}", ${nodeModule}`
```

Comment:
this creates `await await __vitest__importModule__()` isn't it?

Reply:
The transformed code creates the next output:

```javascript
 'await __vitest__mock__("/home/vitest/node_modules/.pnpm/axios@0.24.0/node_modules/axios/index.js", "axios");\n' +
    'const __vite_ssr_import_0__ = await __vite_ssr_import__("/@fs/vitest/packages/vitest/dist/index.js");\n' +
// ...
```

So there are not duplication of `await` key

Reply:
this functions is called for every `vi` function

users run `vi.importActual` with await